scanner/src/funTest/kotlin/storages/ClearlyDefinedStorageFunTest.kt

```diff
@@ -44,10 +48,24 @@ class ClearlyDefinedStorageFunTest : StringSpec({
     val storage = ClearlyDefinedStorage(ClearlyDefinedStorageConfiguration(Server.PRODUCTION.apiUrl))
 
     "Scan results for 'semver4j' from ScanCode should be read correctly" {
-        val id = Identifier("Maven:com.vdurmont:semver4j:3.1.0")
+        val pkg = Package(
+            id = Identifier("Maven:com.vdurmont:semver4j:3.1.0"),
+            declaredLicenses = emptySet(),
+            description = "",
+            homepageUrl = "",
+            binaryArtifact = RemoteArtifact(
+                url = "https://repo1.maven.org/maven2/com/vdurmont/semver4j/3.1.0/semver4j-3.1.0.jar",
+                hash = Hash(
+                    value = "0de1248f09dfe8df3b021c84e0642ee222cceb13",
+                    algorithm = HashAlgorithm.SHA1
+                )
+            ),
+            sourceArtifact = RemoteArtifact.EMPTY,
+            vcs = VcsInfo.EMPTY
+        )
```

Comment:
Use `Package.EMPTY.copy(...)` to set only the relevant properties, same two times below.

Reply:
I deliberately didn't do that to benefit from the default values for `purl` and `vcsProcessed`, but turns out that this was meaningful only in an earlier version of the PR.